backend/src/ml_space_lambda/authorizer/lambda_function.py

```diff
@@ -302,6 +302,8 @@ def lambda_handler(event, context):
             elif requested_resource == "/group" and request_method == "POST":
                 if IS_ADMIN:
                     policy_statement["Effect"] = "Allow"
+            elif requested_resource == "/group_membership_history" and request_method == "GET":
```

Comment:
Is this allowing anyone to get the group membership history?

Reply:
Yes, it is like the other details items. You can pull it if you have access to the group

Reply:
Where is it actually checking that they're a member of this group? Also am I reading this wrong or it it checking for `/group_membership_history` when the path of the lambda is `/group-membership-history` (dashes vs hyphens)?

Reply:
Thank you, I messed it up with my initial deployment, meant to fix it, and forgot 😓

Reply:
I also updated the authorizer to ensure group membership